groups/order-notification-sender/profiles/live-eu-west-2/vars

```diff
@@ -0,0 +1,8 @@
+aws_bucket = "live-eu-west-2.terraform-state.ch.gov.uk"
```

Comment:
Can we get this variable from parameter store or vault. or something ?

Reply:
Thanks Divya; have modified our build scripts accordingly